app/uk/gov/hmrc/plasticpackagingtax/registration/config/ExternalServicesConfig.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.config
+
+import com.google.inject.{Inject, Singleton}
+import play.api.Configuration
+
+@Singleton
+class ExternalServicesConfig @Inject()(val configuration: Configuration) {
+
+  private def loadUrl(key: String): String =
+    configuration.getOptional[String](s"urls.$key").getOrElse(throw new Exception(s"Missing configuration key: urls.$key"))
+
+  val serviceAvailabilityUrl: String = loadUrl("serviceAvailability")
+  val govUkUrl: String = loadUrl("govUk")
+  val organisationsUrl: String = loadUrl("organisationsLink")
+  val explorePpt: String = loadUrl("explorePpt")
```

Comment:
Do we need all these urls?

Reply:
removed govUk as not needed, the other ones I guess they will be needed at some point, serviceAvailability probably will be included on the page, explorePpt is for info on PP tax, organisationsLink is general for hmrc, can removed it for now if not required on start page, what do you think?

Reply:
removed